server.js

```diff
@@ -90,13 +90,13 @@ app.post('/admin', function(req, res) {
 //        Sockets
 ///////////////////////////////////////////////////////////////////////
 
-io.on('connection', (socket) => {  
+io.on('connection', (socket) => {
   // PHASE I
   socket.on('user connect', () => {
-    for (let admin of admins) {
-      socket.broadcast.to(admin).emit('user waiting', socket.id);
-    }
-  }); 
+
+    // console.log('user connect: ' + socket.id)
```

Comment:
I don't think we want this broadcast to every socket. It should be limited to the authenticated admins.